modules/static/protocolbuffers/gofeatures/buf.md

```diff
@@ -1,5 +1,8 @@
 **This is a third-party repository managed by Buf.**
 
+> This module has been **deprecated**. As of v29.2, use the `protocolbuffers/wellknowntypes` module
+> instead.
```

Comment:
I know I suggested this text. But now that I'm reading it, it's not as clear as it could be. My bad 🤦 

It's unclear that v29.2 refers to a version of wellknowntypes, vs. this module. How about this instead:
```suggestion
> This module has been **deprecated**. Use the `protocolbuffers/wellknowntypes` module, v29.2 or higher,
> instead.
```

Reply:
Changed, thanks!